src/uct/ib/base/ib_iface.c

```diff
@@ -369,6 +381,8 @@ UCS_CLASS_INIT_FUNC(uct_ib_iface_t, uct_ib_iface_ops_t *ops, uct_md_h md,
     uct_ib_device_t *dev = &ucs_derived_of(md, uct_ib_md_t)->dev;
     ucs_status_t status;
     uint8_t port_num;
+    cpu_set_t cpu_cores    = worker->worker_params.cpu_cores;
+    int used_irq = find_lcs_cpu(&cpu_cores);
```

Comment:
calling this function should not be done in init part, need to move it to right before calling create cq